packages/quantic/force-app/main/default/lwc/quanticFeedback/__tests__/quanticFeedback.test.js

```diff
@@ -0,0 +1,326 @@
+// @ts-ignore
+import {createElement} from 'lwc';
+import QuanticFeedback from '../quanticFeedback';
+
+const functionsMocks = {
+  exampleHandleLike: jest.fn(),
+  exampleHandleDislike: jest.fn(),
+  exampleHandleExplainWhy: jest.fn(),
+};
+
+const exampleQuestion = 'Example question';
+const exampleSuccessMessage = 'Example success Message';
+
+const selectors = {
+  feedbackQuestion: '.feedback__question',
+  likeButton: '.slds-button:nth-of-type(1)',
```

Comment:
Nitpick: these `:nth-of-type(1)` selectors look a little sketchy given they rely on the positioning of the button.

Reply:
good point 👌 will update the selectors